packages/gatsby-source-contentful/src/fetch.js

```diff
@@ -113,16 +155,24 @@ ${formatPluginOptionsForCLI(pluginConfig.getOriginalPluginOptions(), errors)}`,
     resolveLinks: false,
   }
   try {
+    syncProgress = reporter.createProgress(
+      `Contentful: ${syncToken ? `Sync changed items` : `Sync all items`}`,
+      pageLimit,
+      0
+    )
+    syncProgress.start()
+    reporter.verbose(`Contentful: Sync ${pageLimit} items per page.`)
     let query = syncToken
       ? { nextSyncToken: syncToken, ...basicSyncConfig }
       : { initial: true, ...basicSyncConfig }
     currentSyncData = await client.sync(query)
+    syncProgress.done()
```

Comment:
Could you add this to finally block so we always close the activity.

Reply:
Hah, I wasn't sure whether the panic would just take care of this actually :) So I guess not.